bokeh/layouts.py

```diff
@@ -7,23 +7,23 @@
 #-----------------------------------------------------------------------------
 from __future__ import absolute_import
 
-from .core.enums import Location, Responsive
+from .core.enums import Location, SizingMode
 from .models.tools import ToolbarBox
 from .models.plots import Plot
 from .models.layouts import LayoutDOM, Row, Column, Spacer
 
 
-def layout(children=None, responsive='stretch_both', *args):
+def layout(children=None, sizing_mode='stretch_both', *args):
     """ Create a grid-based arrangement of Bokeh Layout objects. Forces all objects to
-    have the same responsive mode, which is required for complex layouts to work.
+    have the same sizing mode, which is required for complex layouts to work.
 
     Args:
         children List(List(Instance(LayoutDOM))): An list of lists containing any of the
-        following: Plot, Widget, WidgetBox, Row, Column, ToolbarBox, Spacer. All tems in the grid
-        are then assigned the responsive mode of the layout.
+        following: Plot, Widget, WidgetBox, Row, Column, ToolbarBox, Spacer. All items
+        in the grid are then assigned the sizing mode of the layout.
 
-        responsive Enum(``box``, ``fixed``, ``scale_width``, ``scale_height``, ``scale_both``) :  How
-        the grid will respond to the html page. Default is ``box``.
+        sizing_mode ``"fixed"``, ``"scale_width"``, ``"scale_height"``, ``"scale_both"``, and
+        ``"stretch_both"``. Default is ``"stretch_both"``
```

Comment:
I don't like the remove of the explanation of what sizing_mode does. This is one of the primary ways I expect people to use layout and I think more detail is needed. It may not have been a good sentence, but we definitely want a sentence or two.